premailer/premailer.py

```diff
@@ -409,6 +411,18 @@ def transform(self, pretty_print=True, **kwargs):
                 parent = item.getparent()
                 del parent.attrib['class']
 
+        # Add align attributes to images if they have a CSS float value of
+        # right or left. Outlook (both on desktop and the web version) are
+        # bad at parsing CSS (like floats) but they do understand the HTML
```

Comment:
"bad at parsing CSS (like floats)"
That's probably true but that expression is too broad for this particular case. The problem is that they don't understand floats. 
How about phrasing it like "...are bad at understanding CSS floats, but they do under the HTML align attribute." ?